pingouin/multivariate.py

```diff
@@ -254,3 +254,102 @@ def multivariate_ttest(X, Y=None, paired=False):
     stats = {'T2': t2, 'F': fval, 'df1': df1, 'df2': df2, 'pval': pval}
     stats = pd.DataFrame(stats, index=['hotelling'])
     return _postprocess_dataframe(stats)
+
+def box_m (data, vars=None, group=None, alpha=.001):
+    from scipy.stats import chi2
+    """Test equality of covariance matrices.
+
+    Parameters
+    ----------
+    data : :py:class:`pandas.DataFrame`, list or dict
+        Iterable. Can be either a list / dictionnary of iterables
```

Comment:
This is not really clear. Data can be either a long-format Pandas dataFrame, or an iterable (e.g. list, dict), correct?